### PR TITLE
submitting JavaScript-III project

### PR DESCRIPTION
**Pull request by marianahuh**
Submitted JavaScript-III project.

### CODE REVIEW

**Comment by marianahuh** [comment]
Changed out the console.logs to returns.

File: assignments/this.js [hunk 1]
```diff
@@ -1,10 +1,13 @@
 /* The for principles of "this";
 * in your own words. explain the four principle for the "this" keyword below.
-*
-* 1. 
-* 2. 
-* 3. 
-* 4. 
+* 
+* 1. Window binding === if the "this" keywrod is not pointed to a specific use it will always point back to the window object. (Default, Global)
```
Comment (ielvisd) [inline assignments/this.js:4]: Good answers

File: assignments/this.js [hunk 2]
```diff
@@ -13,14 +16,58 @@
...
+
 // Principle 3
 
 // code example for New Binding
 
+function AthleticPerson(athlete) {
+  this.athlete = athlete;
+  this.activity = 'basketball';
+  this.action = function () {
+    console.log(`${this.athlete} is amazing at ${this.activity}`)
```
Comment (ielvisd) [inline assignments/this.js:46]: Good use of the `this` keyword and string template literals

File: assignments/prototypes.js [hunk 3]
```diff
@@ -23,6 +32,18 @@
   * should inherit destroy() from GameObject's prototype
 */
 
+function CharacterStats(charaAttributes) {
```
Comment (ielvisd) [inline assignments/prototypes.js:35]: Great constructor functions and use of the `this` keyword

**Review by ielvisd — APPROVED** [review]
Great work Mariana, good use of the `this` keyword and string template literals

**Pull request merged**